test/setup.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Mocha setup.
+ *
+ */
+
+// The URL you want to be used for the bundle.js when running local tests.
+// The below value is typically used when you are using:
+// "npm run live-serve-build".
+global.bundleUrl = ' https://127.0.0.1:8080/bundle.js';
```

Comment:
This will be so useful to switch out the cdn bundle vs local build during local tests.